run_benchmarks.rb

```diff
@@ -590,6 +591,7 @@ def run_benchmarks(ruby:, ruby_description:, categories:, name_filters:, out_pat
 # Print CSV and PNG file names
 puts
 puts "Output:"
+puts out_json_path
 puts out_tbl_path
```

Comment:
```suggestion
```

It seems too verbose to print both. Can we just print the recommended one? Ideally, `out_tbl_path` should be eliminated too.